src/textual/widgets/_tabs.py

```diff
@@ -590,8 +590,13 @@ def _highlight_active(self, animate: bool = True) -> None:
             tab_region = active_tab.virtual_region.shrink(active_tab.styles.gutter)
             start, end = tab_region.column_span
             if animate:
-                underline.animate("highlight_start", start, duration=0.3)
-                underline.animate("highlight_end", end, duration=0.3)
+
+                def animate_underline() -> None:
+                    """Animate the underline."""
+                    underline.animate("highlight_start", start, duration=0.3)
+                    underline.animate("highlight_end", end, duration=0.3)
+
+                self.set_timer(0.01, lambda: self.call_after_refresh(animate_underline))
```

Comment:
Why don't you just set `delay=0.01` on each call to `animate`?

Reply:
Tried that. Not sure why, but this way reduced the glitch.

Reply:
_Could_ be an issue with the way animations are delayed...